### DISCUSSION TITLE
Opposing Force and Blue Shift compatibility

### DISCUSSION THREAD

**Discussion by moYdaDyr** [sultim-t/xash-rt#197] [Q&A]
Hello guys!
It is great mode, but is it works with Opposing Force and Blue Shift? And if not, do we have way to make this mod compatible with these two addons.
Thanks!

**Comment by blackletum** [comment]
this would be amazing

**Comment by boofiboi** [comment]
I'm working on porting bshift over to the engine.

https://github.com/boofiboi/bshift-xash-rt
